Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPage.mm

```diff
@@ -807,4 +808,25 @@ static void testPerformTextSearchWithQueryStringInWebView(WKWebView *webView, NS
     EXPECT_EQ(overlayCount(webView.get()), 1U);
 }
 
+static void swizzledPerformTextSearchWithQueryString(id, SEL, NSString *, UITextSearchOptions *, id<UITextSearchAggregator> aggregator)
+{
+    [aggregator finishedSearching];
+}
+
+TEST(WebKit, FindInPDF)
+{
+    // Swizzle out the method that performs searching, since PDFHostViewController (a remote view
+    // (controller) cannot be created in TestWebKitAPI, and we cannot actually search the PDF.
+    std::unique_ptr<InstanceMethodSwizzler> isInBackgroundSwizzler = makeUnique<InstanceMethodSwizzler>(NSClassFromString(@"WKPDFView"), @selector(performTextSearchWithQueryString:usingOptions:resultAggregator:), reinterpret_cast<IMP>(swizzledPerformTextSearchWithQueryString));
+
+    RetainPtr<WKWebView> webView = adoptNS([[WKWebView alloc] initWithFrame:NSMakeRect(0, 0, 800, 600)]);
```

Comment:
Nit - `auto` here (or just `RetainPtr`, for consistency with `searchOptions` below)

Reply:
Done. Made this change in the other test too, and used `auto` with `searchOptions` below.